css/home.css

```diff
@@ -0,0 +1,71 @@
+body {
+  background-image: linear-gradient(
+    120deg,
+    var(--bg-bright) 15%,
+    var(--bg-reef) 20%,
+    var(--bg-dark)
+  );
+  color: var(--color-black);
+}
+
+.quiz-container {
+  width: 100vw;
+  margin: 0 auto;
+}
+
+.quiz-content {
+  max-width: 100%;
+  align-items: center;
+}
+
+.content-title {
+  font-family: "Palanquin Dark", sans-serif;
+  font-size: 5rem;
+}
+
+.content-description {
+  width: 40%;
+  letter-spacing: 1px;
+  margin: 3rem 0;
+  font-family: "Lora", serif;
+  font-style: italic;
+  font-size: 1.5rem;
+  text-align: center;
+}
+
+.btn {
+  background-color: var(--color-secondary);
+  display: flex;
+  align-items: center;
+  min-width: 0;
+  width: min-content;
+}
+
+.btn-home {
+  width: auto;
+}
+
+span {
+  color: var(--color-white);
+  letter-spacing: 0px;
+  font-size: 2rem;
+  transition: all 0.1s ease-in;
+}
+
+span:hover {
```

Comment:
avoid giving styling directly to the elements

Reply:
Sure, will be giving class names to avoid this in future.